src/org/labkey/targetedms/chromlib/Constants.java

```diff
@@ -710,8 +707,8 @@ public String definition()
     {
         Id(Column.Id),
         MoleculeListId(Column.MoleculeListId),
-        IonFormula(Column.IonFormula),
-        CustomIonName(Column.CustomIonName),
+        IonFormula(Column.ChemicalFormula),
+        CustomIonName(Column.MoleculeName),
```

Comment:
may be use Column.FragmentName

Reply:
This pattern's a little clunky, but the two names should agree with each other. I found a couple of cases that were out of sync, including this one.